### PR TITLE
Created a createUser function

### PR DESCRIPTION
**Pull request by CastroDan**

### CODE REVIEW

File: db/users.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'use strict'
```
Comment (renodubois) [inline db/users.js:1]: Don't think this is needed, but @michael-beaver might be able to shed more light on that.

**Review by renodubois — APPROVED** [review]
Minus the one comment I left, this seems to be pretty good.

File: db/users.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const User = require('./init').User
+
+function createUser (username, password, email) {
```
Comment (michael-beaver) [inline db/users.js:5]: You should check to see that username, password, and email are not null or undefined

File: db/users.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  return new Promise((resolve, reject) => {
+    const newUser = new User(
+      {
+        username: username,
+        password: password,
+        email: email
+      })
```
Comment (michael-beaver) [inline db/users.js]: You need to include fields that do not have default values specified. For example, the discordID field. In this case probably go ahead and just initialize them to empty strings where appropriate

File: db/users.js (hunk 1 above)
Comment (michael-beaver) [reply db/users.js:1]: Yeah apparently this isn't necessary anymore with ECMAScript 6

**Review by michael-beaver — CHANGES_REQUESTED** [review]

**Review by michael-beaver — APPROVED** [review]

**Pull request merged**